File: eRent.Model/Korisnik.cs
using System;
using System.Collections.Generic;

namespace eRent.Model
{
    public class Korisnik
    {
        public string KorsnikIme { get; set; }

        public string KorisnikPrezime { get; set; } 
           
        public string Email { get; set; }

        public int? UlogaId { get; set; }

        public double? Rejting { get; set; }
    }
}

File: mobile/myapp/lib/model/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  late int? korisnikId;
  late String? korisnikIme;
  late String? username;
  late String? korisnikPrezime;
  late String? email;
  late int? ulogaId;
  late int? rejting;

  Korisnik();

  factory Korisnik.fromJson(Map<String, dynamic> json) =>
      _$KorisnikFromJson(json);

  /// Connect the generated [_$ProductToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}
